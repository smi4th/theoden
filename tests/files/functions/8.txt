fonction a(arg1, arg2) {
    return arg1 + arg2;
}
v = 1;
resp = a(v * 2, v * 3);