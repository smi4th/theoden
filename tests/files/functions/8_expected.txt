---TEST OUTPUT---
---MEMORY---
v = 1
resp = 5
---FUNCTIONS---
a = (('param', ('param', 'arg1'), 'arg2'), ('linst', ('return', ('+', 'arg1', 'arg2'))))
